src/utils/ip.ts

```diff
@@ -40,3 +44,19 @@ export default function getIp(): string {
 
   return address
 }
+
+export function getPublicIp() {
+  axios({
+    method: 'GET',
+    url: 'https://ifconfig.co/ip',
+  })
+    .then((res) => {
+      if (res?.data) {
+        log.info(`Monika is running on Public IP ${res.data}`)
+        publicIpAddress = res.data
+      }
+    })
+    .catch(() => {
+      log.info(`Can't obtain Public IP`)
+    })
```

Comment:
I think we should use [stun](https://www.npmjs.com/package/stun) instead to get the IP address.

Reply:
Implemented